pkg/storaged/client.js

```diff
@@ -188,6 +188,7 @@ function init_proxies () {
     client.vdo_vols = proxies("VDOVolume");
     client.blocks_fsys_btrfs = proxies("Filesystem.BTRFS");
     client.jobs = proxies("Job");
+    client.nvme_controller = proxies("NVMe.Controller");
```

Comment:
This is new since 2.10.0 so won't be available on RHEL-9-6 which has 2.9.4, is this gracefully handled?

Reply:
should be, even for a nonexisting iface it doesn't fail and returns a generic dbus endpoint. It doesn't have any `drive.path` related props so it won't get through on the following check
`    } else if (client.nvme_controller[drive.path]) {`